tests/data/example_config.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "test_run",
+    "user_id": "test_user1@mail.com",
+    "job_id": "BaxDa",
+    "baseurl": "http://localhost:3000",
```

Comment:
why do you need to store this here? I think this should be part of the API config not the job config.

Reply:
I made this config file as mock data for one of the test, but I realized that I can just store an empty JobConfig object so I'll just remove this file.

But I will keep in mind your comments here and below in picking which fields gets stored on disk using the `from_json` function.

I have another PR that I'm working on that changes the status messages returned by the API to return the request rather than the job config.

Reply:
I see, what should we do? merge this and review the other PR?

Reply:
That other PR can wait until we merge both of the Redis PRs, I don't think it a priority